src/main/java/Storage.java

```diff
@@ -0,0 +1,200 @@
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Storage {
+    String filePath;
+
+    /**
+     * Constructor for Storage class.
+     *
+     * @param filePath The filepath to get the Hard Disk data from.
+     * @throws IOException Throws exception.
+     */
+    public Storage(String filePath) throws IOException {
+        // deals with loading tasks from the file and saving tasks in the dukeStorage.txt file.
+        this.filePath = filePath;
+        File file = new File(filePath);
+        file.createNewFile();
+    }
+
+    /**
+     * Load hard disk information into an ArrayList for TaskList class to use.
+     *
+     * @return Returns an ArrayList of Task, loaded from the Hard Disk.
+     * @throws FileNotFoundException Throws exception.
+     * @throws GrapieExceptions Throws exception.
+     */
+    public List<Task> load() throws FileNotFoundException, GrapieExceptions {
+        List<Task> storingList = new ArrayList<>();
+
+        File myObj = new File(filePath);
+        Scanner myReader = new Scanner(myObj);
+        while (myReader.hasNextLine()) {
+            String data = myReader.nextLine();
+
+            /*
+            T | 1 | read book
+            D | 0 | return book | June 6th
+            E | 0 | project meeting | Aug 6th 2-4pm
+            T | 1 | join sports club
+            */
+
+            String[] dataSplited = data.split("\\|");
+
+            //trim off empty spaces at front and back of string
+            for (int i = 0; i < dataSplited.length; i++) {
+                dataSplited[i] = dataSplited[i].trim();
+            }
+            //System.out.println(dataSplited.length);
+
+            if (dataSplited.length == 3) {
+                //is a todo
+                Task task = new Todo(dataSplited[2]);
+                if (dataSplited[1].equals("O")) {
+                    task.isDone = true;
+                }
+                storingList.add(task);
+
+            } else if (dataSplited.length == 4) {
+                //is a event or deadline
+                if (dataSplited[0].equals("E")) {
```

Comment:
Actually, since the tasks have character ids, you could rework load to use a switch-case statement instead.

eg.
case "E":
    if (dataSplitted.length == 4) ...